src/registerAdapters.ts

```diff
@@ -31,6 +32,12 @@ function registerAdapterForServer(serverName: string, server: Server, app: Glee,
       server,
       parsedAsyncAPI,
     })
+  } else if (['kafka'].includes(protocol)) {
```

Comment:
You're not handling the `kafka-secure` protocol yet but you will so let's better add it now:

```suggestion
  } else if (['kafka', 'kafka-secure'].includes(protocol)) {
```